Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openshift/origin-release:golang-1.13 AS builder
```

Comment:
we have 

`FROM rhel8/go-toolset:1.13.4 AS builder` in our downstream builds.

and @rhuss has that in here too:
https://github.com/openshift/knative-client/blob/master/Dockerfile.rhel#L1

Reply:
updated to that